docs/src/PageHeader.doc.js

```diff
@@ -0,0 +1,643 @@
+                )}
+              </React.Fragment>
+            }
+            primaryAction={<Button color="red" size="lg" text="Create product group" />}
+          />
+        );
+      }
+    `}
+  />,
+);
+
+card(
+  <PropTable
+    Component={PageHeader}
+    props={[
+      {
+        name: 'title',
+        type: 'string',
+        required: true,
+        description:
+          'Page title, will always be a level 1 heading. Content should be [Localized](/Localized).',
+      },
+      {
+        name: 'maxWidth',
+        type: `number | string`,
+        description: `Use numbers for pixels: \`maxWidth={100}\` and strings for percentages: \`maxWidth="100%"\`.`,
+      },
+      {
+        name: 'primaryAction',
+        type: `Element<typeof Button | typeof IconButton | typeof Link | typeof Tooltip>`,
+        description: `The primary action of the page. Can be a [Button](/Button), [IconButton](/IconButton), [Link](/Link), [Tooltip](/Tooltip) surrounding an IconButton or a combination of an IconButton, Tooltip, and [Dropdown](/Dropdown).`,
+      },
+      {
+        name: 'secondaryAction',
+        type: `Element<typeof Button | typeof IconButton | typeof Link | typeof Tooltip>`,
+        description: `A secondary action for the page. Can be a [Button](/Button), [IconButton](/IconButton), [Link](/Link), [Tooltip](/Tooltip) surrounding an IconButton or a combination of an IconButton, Tooltip, and [Dropdown](/Dropdown).`,
+      },
+      {
+        name: 'subtext',
+        type: `string`,
+        description: `Used for metadata related to the current page, not designed to describe the title or the current surface. Content should be [Localized](/Localized).`,
+      },
+    ]}
+  />,
+);
+
+card(
+  <MainSection name="Best practices">
+    <MainSection.Subsection>
+      <MainSection.Card
+        cardSize="lg"
+        type="do"
+        description="Use only one primary style action in PageHeader ."
+        defaultCode={`
+<PageHeader
+  title="Product groups"
+  subtext="S. E. All products USD"
+  primaryAction={<Button color="red" size="lg" text="Create group" />}
+/>;
+        `}
+        shaded
+      />
+      <MainSection.Card
+        cardSize="lg"
+        type="do"
+        description={`
+        Ensure the title of PageHeader matches the title of the item that navigated the user to this page. For instance, if the user selects "Settings" from an overflow menu, the title of PageHeader should also say "Settings".
+        `}
+        defaultCode={`
+<PageHeader
+  title="Settings"
+  primaryAction={
+    <Tooltip text="Additional options">
+      <IconButton icon="ellipsis" iconColor="darkGray" size="lg" accessibilityLabel="" />
+    </Tooltip>
+  }
+/>;
+        `}
+        shaded
+      />
+      <MainSection.Card
+        cardSize="lg"
+        type="do"
+        description={`
+        Plan for most PageHeaders to be full width. A \`maxWidth\` should only be supplied when the content of the page is center aligned.
```

Comment:
What's the difference in this example from the previous ones? I'm not sure I'm connecting the text with the example.

Reply:
I think I got it from looking at the Variant. It's a tricky one.

Reply:
nit comment: I'd replace texfield with name, address, etc.